docs/testing.md

```diff
@@ -320,8 +320,7 @@ automatically in Zulip's Travis CI tests to avoid any regressions, but
 is not included in test-all since it is quite slow.
 
 To run tooks/check-py3, you need to install the modernize and future
-python packages (which are in the development environment's
-requirements.txt file).
+python packages (which are in requirements/dev.txt).
```

Comment:
Yes. But it's also indirectly in requirements/dev.txt.

I thought the main reason to mention that modernize and future were in requirements.txt was that people who have done `pip install -r requirements.txt` would know that they already have it. Most people will not manually do `pip install -r requirements/py3k.txt`. They would instead do `requirements/dev.txt`. So I thought it should be requirements/dev.txt.

But I think the best solution would be to mention both and explain that py3k.txt is included in dev.txt.